repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -278,6 +278,11 @@ export interface PremadeTimeRestriction {
     contents: DbTimeRestriction[];
 }
 
+export interface PremadeCap {
```

Comment:
What is meant by a PremadeCap? As far as I know we only have one type of Cap, and its the ones they make in operator settings.